packages/client/src/tests/util.tsx

```diff
@@ -45,10 +45,14 @@ import { stringify } from 'query-string'
 import { match as Match } from 'react-router'
 import { ConnectedRouter } from 'connected-react-router'
 import { mockOfflineData } from './mock-offline-data'
-import { SubmissionAction } from '@client/forms'
+import { Section, SubmissionAction } from '@client/forms'
```

Comment:
Changes in this file don't seem like ones we really want to be making to develop as they are not related. Could we remove these?

Reply:
If we remove that, it will affect this code
`deserializeFormSection({
            id: 'user' as Section,
            viewType: 'form',
            name: {
              defaultMessage: 'User',
              description: 'The name of the user form',
              id: 'constants.user'
            },`
            @rikukissa

Reply:
@modupeadeonojobi okay no worries! I don't know why this file was showing in the diff before as these aren't changes that we would have intentionally added here 🤔